SPECS/SymCrypt-OpenSSL/SymCrypt-OpenSSL.spec

```diff
@@ -13,6 +13,7 @@ BuildRequires:  SymCrypt
 BuildRequires:  cmake
 BuildRequires:  gcc
 BuildRequires:  make
+ExclusiveArch:  x86_64
```

Comment:
This and same change inside `SymCrypt.spec` seem unrelated to the topic. Bad merge? If we do need it, let's also increment the release and add a changelog entry.

Reply:
Huh, I wonder why this ended up as part of the PR when I am based after that PR was merged..